db/interface.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding:utf-8 -*-
+
+from db.models import Payment, Detail
+
+
+class PaymentService(object):
+    """
+    Serves as an interface which takes requests from the front-end
+    and runs the corresponding actions in the database.
+    """
+
+    def __init__(self):
```

Comment:
UPDATE: i've been messing around with this and have a working version after reorganizing the project a bit. i created a new branch (interface-review) to experiment with and have just pushed up here for anyone ( @jdgumz @SharanyaT @nvme5 ) to take a look, but here's the short of it:

-created an app module that instantiates app and db. this module imports interface, models, payments, which all reside in app folder.
-pulled out the main function into a separate run script.
-interface init takes db as param for internal use.-wrote a crude add_payments just to test out functionality and it seems to work thus far, but let me know what you think w/r/t design/architecture.